frontend/src/components/user-details/components/user-profile/components/user-profile-form/select.data.ts

```diff
@@ -0,0 +1,9 @@
+import { UserGender } from 'common/enums/enums';
+import { SelectorOptions } from 'common/types/types';
+
+export const SelectData: SelectorOptions[] = Object.values(UserGender).map(
```

Comment:
Let's make export at the bottom - and I think something like `genderOptions` would be a better naming

Reply:
agreed. you can do it in file of the component